### PR TITLE
0.2

### PR DESCRIPTION
**Pull request by mirii1994**
This PR:
* Allow parsing log into fields, as raised by #9 
* Allow parsing a log that's a JSON into nested fields.
* Adds explanation about the above features to the readme (still a work in progress ~needs edit from the ux team~)

### CODE REVIEW

File: logzio-lambda-extensions-logs/go.mod [hunk 1]
```diff
@@ -7,4 +7,5 @@ require (
 	github.com/logzio/logzio-go v1.0.2
```
Comment (Doron-Bargo) [inline logzio-lambda-extensions-logs/go.mod:7]: Please upgrade to version 1.0.3

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Great work
Please see my comment
Also please add negative and positive tests for the Grok option

**Review by Doron-Bargo — APPROVED** [review]
Great work on the positive and negative tests

**Comment by Doron-Bargo** [comment]
@YajB - can you check the Readme?

**Comment by Doron-Bargo** [comment]
@nico-shishkin - can you go over the readme?

**Review by nico-shishkin — APPROVED** [review]

**Pull request merged**